open-machine-learning-jupyter-book/deep-learning/GAN.md

```diff
@@ -13,5 +13,267 @@ kernelspec:
   name: python3
 ---
 
-# GAN (TBD)
+# Generative adversarial networks 
 
+The original purpose of GAN is to generate new data. It classically generates new images, but is applicable to wide range of domains.
+It learns the training set distribution and can generate new images that have never been seen before.
+In contrast to e.g., autoregressive models or RNNs (generating one word at a time), GANs generate the whole output all at once.

```

Comment:
We must be consistent for the name. We will use GANs or GAN?

Reply:
Done.